Remove ``_warnings.py`` and all uses - related Cython issue is resolved
